### ISSUE TITLE
Low gamma-glutamyltransferase level

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5948]
**Preferred term label:** Low gamma-glutamyltransferase level

**Synonyms**

**Definition (free text, please give PubMed ID)** Decreased level of the enzyme gamma-glutamyltransferase (GGT)., PMID:31455253

**Parent term (use hpo.jax.org/app)** Abnormal enzyme/coenzyme activity HP:0012379

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
There is no medically relevant "low" GGT level -- basically, the test is whether GGT is increased or not.
